src/js/app.js

```
@@ -1281,6 +1284,23 @@ if (typeof document !== 'undefined' && document.addEventListener) {
   }, true);
 }
 
+function updateClearAllButtonVisibility() {
+  const clearBtn = document.getElementById('clearAllFilters');
+  if (!clearBtn) return;
+
+  const hasActiveFilters = 
+    (typeof selectedLanguages !== 'undefined' && selectedLanguages.size > 0) ||
+    (typeof activeChip !== 'undefined' && activeChip !== null) ||
+    (typeof activeFilters !== 'undefined' && activeFilters.length > 0) ||
```

ESLint

[error] 'activeFilters' is not defined.

(no-undef)